packages/component-library/src/Scatterplot/Scatterplot.js

```diff
@@ -186,6 +84,7 @@ const Scatterplot = ({
         <VictoryAxis
           animate={{ onEnter: { duration: 500 } }}
           style={{ grid: { stroke: 'none' } }}
+          tickFormat={x => xNumberFormatter(x)}
```

Comment:
you can instead just pass the formatter in directly: `tickFormat={xNumberFormatter}`.

Reply:
Sweet, will do 👍